### DISCUSSION TITLE
Huami OS Simulator - apps do not appear

### DISCUSSION THREAD

**Discussion by SantoCovato** [zepp-health/zeppos-forum#357] [Q&A]
I followed the instructions on the zepp OS blog to create an application project but when I enter Huami OS Emulator, no application appears. What can I do to resolve it?
![Screenshot 2024-07-03 150100](https://github.com/zepp-health/zeppos-forum/assets/104246135/e6a16372-6906-4378-905a-5985ec5be1a4)
![Immagine 2024-07-03 150134](https://github.com/zepp-health/zeppos-forum/assets/104246135/d86487f1-c6a0-4110-b478-3baa612f844d)

**Comment by silver-zepp** [comment]
Hey @Santix29, you seem to be missing a couple of steps:
1. you have to make sure that you are `logged in`
2. click on the `Emulator` button, to start the actual machine
3. type `zeus dev` in the separate console

Have a look at this tutorial to know more:
https://www.youtube.com/watch?v=RPyTge0kjWQ

**Reply by silver-zepp** [reply]
Also, ensure you don't keep your projects in a Windows folder. Otherwise, you'd have to run your whole toolchain with Admin rights.

**Reply by SantoCovato** [reply]
I had already done these steps before (sorry for not mentioning it), now I tried to create a project in a non-Windows folder, but the situation is still the same.
![Immagine 2024-07-03 160211](https://github.com/zepp-health/zeppos-forum/assets/104246135/797a2725-ecb8-498a-a273-2ff8d6c9f0f9)

**Reply by silver-zepp** [reply]
Thank you for the additional screenshots. I believe the issue is that you have downloaded Emulators in the past and they are incompatible with Simulator 2.0. 

Try this: 
1. close any opened Emulators
2. open up the Emulator download button (cloud)
3. click on the "Patch downloaded" button
4. wait for the process to complete and start the Emulator again
5. `zeus dev`
![image](https://github.com/zepp-health/zeppos-forum/assets/138894963/cca91775-dfda-4ac0-8e0d-66bb6e7e5d86)

If that doesn't help, make sure you run your shell without administrative rights.

**Reply by SantoCovato** [reply]
I tried the steps you wrote to me, when I ran it as administrator nothing changed, while when I ran it normally I got this error:
![Immagine 2024-07-03 163136](https://github.com/zepp-health/zeppos-forum/assets/104246135/b2356e90-fff4-4001-aa53-3a6d3bb76c70)
I also noticed another thing, I don't know if it has anything to do with it but I saw that when I click the login button, that is this:
![Immagine 2024-07-03 163159](https://github.com/zepp-health/zeppos-forum/assets/104246135/f319e6eb-2247-4d2f-bfc6-e9bf1a49b152)
it asks me again to enter my email and password even though I have done so previously, is this normal?

**Reply by zeppos-hx** [reply]
The error that reported was a user permission issue, because you created this folder as administrator, so the normal user has no permission to delete it, could you please create another project folder and try it again?

**Reply by SantoCovato** [reply]
Ok, now I created the folder without the administrator permission, but still no projects on the simulator.

**Reply by zeppos-hx** [reply]
OK, I think you should click the avatar icon to login first, and then run `zeus dev` to push mini program to the simulator, if there is still nothing to shown, please share me what you see on the `Console` panel

**Reply by SantoCovato** [reply]
I tried now, but nothing changed. That's the console panel: 
![Immagine 2024-07-10 122040](https://github.com/zepp-health/zeppos-forum/assets/104246135/9910c601-a03c-46b6-9e11-c0cad95479e5)
